### DISCUSSION TITLE
osu! Portable support ?

### DISCUSSION THREAD

**Discussion by Schweeeeeeeeeeeeeeee** [ppy/osu#17706] [Ideas]
I've been trying to make osu! lazer portable but I'm just not smart enough it would be really nice because then u can plug a USB in anybody's pc and play with your saves without needing to download everything again.

The Game already is portable, but not the saves beat maps and all of that.

**Comment by peppy** [comment]
~~For now, please make a `framework.ini` in the same place as `osu!.exe`.~~

We'll likely add support for this in the future based on demand.

**Reply by Joehuu** [reply]
For anyone looking at this, this was unsupported when it was said, but this mustn't be done anymore (after the switch to Velopack, the update manager).

Ref: https://osu.ppy.sh/community/forums/topics/2187136?n=6

**Comment by Nekonyx** [comment]
+1
osu! is probably the only game that I constantly archive and transfer from my old hard drive to a new one. I've been doing this since almost 2016.
